test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-
+docker pull registry.tld/ruby-fips-base-image-phusion:1.0.0
```

Comment:
ubuntu ?

Reply:
This should be removed when phusion will publish to dockerhub